examples/notebooks/DynamicBalanceClassSampler-example.ipynb

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
let's separate this example from the Catalyst notebooks examples and just place it as a dedicated colab one.
I mean, let's just crate a colab and share the link into the docs.